pitop/pma/encoder_motor.py

```diff
@@ -62,6 +68,21 @@ def __init__(self,
 
         atexit.register(self.stop)
 
+        Stateful.__init__(self)
+        Recreatable.__init__(self, config_dict={'port_name': port_name, 'name': name})
+        # Added as lambdas since they may change on runtime
+        self.add_to_config("forward_direction", lambda: self.forward_direction)
+        self.add_to_config("braking_type", lambda: self.braking_type)
+        self.add_to_config("wheel_diameter", lambda: self.wheel_diameter)
+
+    @property
+    def own_state(self):
+        return {
+            'current_rpm': lambda: self.current_rpm,
+            'current_speed': lambda: self.current_speed,
+            'distance': lambda: self.distance,
```

Comment:
There's quite a bit more useful information that could be provided here. Braking type? Wheel diameter?

Reply:
Or perhaps that is more "config" rather than "state. But I wonder why we wouldn't include that as well...

Reply:
agreed!